agenta-web/src/lib/helpers/axiosConfig.ts

```diff
@@ -20,7 +20,13 @@ axios.interceptors.response.use(
             }`
         error.message = msg
 
-        globalErrorHandler(error)
+        const apiCallUrl: string = error.response.request.responseURL
```

Comment:
Please don't handle individual API responses here. This is a a global axios config and meant to be generic. If you don't want errors of any api to be handled automatically, then you can use the `_ignoreError` attribute in the config of that api. You can find some examples of this in the `api.ts` file